docs/deployment/uwsgi_cookiecutter_nginx.rst

```diff
@@ -0,0 +1,233 @@
+    the name of the file, ``wsgi`` when we invoke uWSGI.
+
+    The call to ``loader.setup_logging`` initializes the standard library's
+    :mod:`python3:logging` module through :func:`pyramid.paster.setup_logging`
+    to allow logging within your application. See
+    :ref:`pyramid:logging_config`.
+
+#.  Create a new directory at ``~/myproject/tmp`` to house a pidfile and a unix
+    socket.  However, you'll need to make sure that *two* users have access to
+    change into the ``~/myproject/tmp`` directory: your current user (mine is
+    ``ubuntu`` and the user that Nginx will run as often named ``www-data`` or
+    ``nginx``).
+
+#.  Invoke uWSGI.
+
+    .. code-block:: bash
+
+        cd ~/myproject
+        sudo uwsgi \
+          --chmod-socket=020 \
+          --enable-threads \
+          --plugin=python3 \
+          --socket ~/myproject/tmp/myproject.sock \
+          --manage-script-name \
+          --mount /=wsgi:app \
+          --uid ubuntu \
+          --gid www-data \
+          --virtualenv env
+
+        # Explanation of Options
+        # sudo uwsgi                          # Invoke as sudo so you can masquerade
+        #                                       as the users specfied in --uid and --gid
+        #
+        # --chmod-socket=020                  # Change permissions on socket to
+        #                                       at least 020 so that in combination
+        #                                       with "--gid www-data", Nginx will be able
+        #                                       to write to it after  uWSGI creates it
+        #
+        # --enable-threads                    # Execute threads that are in your app
+        #
+        # --plugin=python3                    # Use the python3 plugin
+        #
+        # --socket ~/myproject/tmp/myproject.sock   # Where to put the unix socket
+        #
+        # --manage-script-name
+        #
+        # --mount /=wsgi:app                  # Mount the path "/" on the symbol
+        #                                       "app" found in the file wsgi.py
+        #
+        # --uid ubuntu                        # masquerade as the ubuntu user
+        #
+        # --gid www-data                      # masquerade as the www-data group
+        #
+        # --virtualenv env                    # Use packages installed in your venv
+
+#.  Verify that the output of the previous step includes a line that looks approximately like this:
+
+    .. code-block:: bash
+
+        WSGI app 0 (mountpoint='/') ready in 1 seconds on interpreter 0x5615894a69a0 pid: 8827 (default app)
+
+    If any errors occurred, you will need to correct them. If you get a
+    ``callable not found or import error``, make sure you your ``--mount /=wsgi:app``
+    matches the ``app`` symbol in the ``wsgi.py`` file. An import
+    error that looks like ``ImportError: No module named 'wsgi'`` probably
+    indicates a mismatch in your --mount arguments. Any other import errors
```

Comment:
```
your ``--mount`` arguments
```

Reply:
@stevepiercy the ``--mount`` is actually the phrase 

    ``--mount /=wsgi:app``

I have moved it to the next line so it does not wrap on the narrow github display. If you were wanting something different, let me know. 

Other than that, PR is ready to merge.

Reply:
--mount must be surrounded by double-backticks so that it is displayed properly when rendered. Otherwise it is displayed as a plain old English word, which it ain't.